.env.sample

```diff
@@ -17,9 +17,11 @@ PLAID_LANGUAGE="en"
 
 ACTUAL_SERVER_URL="server url"
 ACTUAL_SERVER_PASSWORD="server password"
+ACTUAL_SERVER_ENCRYPTION_PASSWORD="server password"
```

Comment:
Wont this example value cause the script to always think that budget file is encrypted?  I think it should either be blank by default, or there should still be a boolean for encrypted.